packages/flutter_tools/test/general.shard/resident_runner_test.dart

```diff
@@ -423,6 +423,11 @@ void main() {
     expect(testLogger.errorText, contains('Error'));
   }));
 
+  test('ResidentTunner clears the screen when it should', () => testbed.run(() async {
+    residentRunner.clearScreen(testLogger);
+    expect(testLogger.statusText, equals(''));
+  }));
```

Comment:
I believe the logger is already the testLogger here, so you shouldn't need to provide it at all

Reply:
Though you should assert that the BufferLogger clear is called somehow

Reply:
About asserting the call is actually made, shouldn't `expect(testLogger.statusText, equals(''));` be enough? After all, that's the practical effect we want to achieve.

Reply:
but was there anything in the logger before you cleared it?

Reply:
You're right, `testLogger` was previously empty. I added a new commit which first writes a message and then checks that it was there before being cleared, and that it's gone after.